torch/_torch_docs.py

```diff
@@ -5590,6 +5590,28 @@ def merge_dicts(*dicts):
     tensor([ 1., -1.,  0.,  1.])
 """.format(**common_args))
 
+add_docstr(torch.signbit,
+           r"""
+signbit(input, out=None) -> Tensor
+
+Returns a new tensor with the signbits of the elements of :attr:`input`.
+
+.. math::
+    \text{out}_{i} = \operatorname{sgn}(\text{input}_{i})
+""" + r"""
+Args:
+    {input}
+    {out}
+
+Example::
+
+    >>> a = torch.tensor([0.7, -1.2, 0., 2.3])
+    >>> a
```

Comment:
This is a good example but you don't need to print 'a', it's clear from your construction what its values are.

Reply:
Addressed, the redundant command has been deleted.